packages/dd-trace/test/opentracing/propagation/text_map.spec.js

```diff
@@ -233,6 +234,38 @@ describe('TextMapPropagator', () => {
 
       expect(carrier).to.not.have.property('x-b3-traceid')
     })
+
+    it('should skip injection of traceparent header without the feature flag', () => {
+      const carrier = {}
+      const spanContext = createContext({
+        traceId: id('0000000000000123'),
+        spanId: id('0000000000000456')
+      })
+
+      config.tracePropagationStyle.inject = []
+
+      propagator.inject(spanContext, carrier)
+
+      expect(carrier).to.not.have.property('traceparent')
```

Comment:
The injected header is called `traceparent` while the config is called `tracecontext`. The reason for this difference is because it is _also_ meant to configure presence of the `tracestate` header too, and the spec itself is called W3C TraceContext.